templates/example_flowsom_clustering.ipynb

```diff
@@ -15,6 +15,7 @@
    "source": [
```

Comment:
<p>Is this the best way to call R?  If it's possible, it might be better to hide away our calls to R within python functions (keep notebooks all python).  Disregard this comment if you're just using this for debugging/dev.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
@ackagel for now it's just for debugging (you'll see that I also have a ton of `sys.path.append`'s as well, those gotta go too). The `!` makes it super easy for now to call `Rscript` inside the Docker, but we can absolutely move that to a `subprocess` call inside a function in `phenotyping`.